project/plugins.sbt

```diff
@@ -1,4 +1,5 @@
-addSbtPlugin("me.lessis" % "bintray-sbt" % "0.3.0")
+addSbtPlugin("me.lessis"    % "bintray-sbt" % "0.3.0")
+addSbtPlugin("com.jsuereth" % "sbt-pgp"     % "1.0.0")
```

Comment:
Yep, previously sbt-pgp wasn't included in the build by default (i.e. no `publishSigned`.)